Program.cs

```diff
@@ -0,0 +1,36 @@
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+
+// Configure swagger to use strings for enums
+builder.Services.AddSwaggerGen(config => { config.UseInlineDefinitionsForEnums(); });
+builder.Services.Configure<Microsoft.AspNetCore.Http.Json.JsonOptions>(options =>
+{
+    options.SerializerOptions.Converters.Add(new System.Text.Json.Serialization.JsonStringEnumConverter());
```

Comment:
Yeah, System.Text.Json is already configured here, so shouldn't mix-and-match with Newtonsoft elsewhere or can just remove that from the dependency if it's not used.

Reply:
Fixed